composer.json

```diff
@@ -13,10 +13,10 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "guzzle/guzzle": "^2.8|^3.0",
+        "guzzlehttp/guzzle": "^6.0",
```

Comment:
This constraint will allow [any version above 6.0](http://semver.mwl.be/#?package=guzzlehttp%2Fguzzle&version=%5E6.0&minimum-stability=stable). I think this is sufficient.